### PR TITLE
add 'automatic' translations

### PR DESCRIPTION
**Pull request by djwester**

### CODE REVIEW

**Comment by djwester** [comment]
Most of the changes are just alphabetizing.  I think I cleaned up all the translators messes

File: build/lang/de.js [hunk 1]
```diff
@@ -1,136 +1,136 @@
...
 		'changeScoringSuccessful': 'Bewertungsmethode zu {method} geändert',
-		'clearOverride': 'Außerkraftsetzung aufheben',
 		'clearFeedback': 'Feedback löschen',
+		'clearOverride': 'Außerkraftsetzung aufheben',
+		'closeDialog': 'Schließen',
 		'criterionAdded': 'Ein neues Kriterium wurde hinzugefügt',
-		'criterionAriaLabel': 'Kriterium {index, number} von {count, number}',
-		'criterionDeleted': 'Kriterium {name} gelöscht',
-		'criterionDescriptionAriaLabel': 'Beschreibung für Kriterium {criterionName}, Stufe {levelName}',
+		'criterionAriaLabel': 'Criterion {index, number} of {count, number}',
```
Comment (sarahboyd) [inline build/lang/de.js]: should this be adding back the english translation?

File: build/lang/en.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ D2L.PolymerBehaviors.Rubric.LocalizeBehavior.LangEnBehavior = {
 		'pointsAreRequired': 'Point value is required',
 		'pointsMinimum': '{number} {number, plural, one {point} other {points}} minimum',
 		'pointsSaveFailed': 'Saving points failed',
-		'percentage': '\{number\} %',
+		'percentage': '{number} %',
```
Comment (sarahboyd) [inline build/lang/en.js]: was this intentional?

File: build/lang/de.js (hunk 1 above)
Comment (djwester) [reply build/lang/de.js]: No, in general in the file with German translations one wants German. But the translators; they know best.

File: build/lang/en.js (hunk 2 above)
Comment (djwester) [reply build/lang/en.js]: yeah, I think it's fine - I don't know why we did that int eh first place

File: build/lang/de.js (hunk 1 above)
Comment (djwester) [reply build/lang/de.js]: I think I've got them all back to German now

**Pull request merged**